free-2.md

```diff
@@ -35,10 +61,27 @@ user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
 
+Once events and TODOs are in, getting the information out should also be easy.
+Ideally, there will be some levels of reminders that go from "this is what you
+should take care of today" to "you have to start this assignment now or else
+you'll skip the talk you really want to go to later."
+
+I feel this is the right way because it's how my brain works. When I see a new
+thing I have to do, I only think of it for so long before it leaves my mind so
+I want to get it down as soon as possible. I also am often asking "Do I have
+enough free time to play a few games?" or "I'm feeling productive, what do I
+need to do now?" This system could solve both.
+
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to add both events with defined starting and ending points
+and tasks that have to get done and have an approximate length but no specified
+time frame. Having some other type of event might be possible but difficult,
+although I don't know of anything else necessary. Nothing in the domain should
+really be impossible since the language should be generic enough to support
+basically any tasks.
```

Comment:
You would probably also want capabilities on par with things like Gantt charts, if feasible. Todoist users have been screaming for that for a long time. Either that, or, allowing such things to be implementable user-side.